vignettes/publisher_vignette.Rmd

```diff
@@ -0,0 +1,122 @@
+---
+title: "fulltext Vignette: Additional Publishers"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{fulltext Vignette: Additional Publishers}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
+---
+
+## About the package
```

Comment:
You don't need to introduce the package again, just focus on the topic of the vignette